polaris.shopify.com/src/components/TokensPage/TokensPage.tsx

```diff
@@ -9,14 +9,13 @@ import Page from '../Page';
 
 interface Props {
   tokenGroup:
+    | 'border'
     | 'breakpoints'
-    | 'colors'
-    | 'depth'
+    | 'color'
     | 'font'
     | 'motion'
     | 'shadow'
-    | 'shape'
-    | 'spacing'
+    | 'space'
```

Comment:
```suggestion
    | 'space'
```
Revert handled in Alex's PR